SEED-platform/seed
# Portfolio summary move to backend

#### Any background context you want to provide?
ICF reported sorting issues on the goal_note columns, leading to duplicate and unsorted results.

#### What's this PR do?
- Moves many of the portfolio summary formatting functions to the backend. The property retrieval function was originally reused from the inventory list, however as the portfolio summary has grown in complexity, having a dedicated function for property retrieval will simply the requests.
- Fixes the portfolio summary sorting on goal_note columns (question, passed checks...). The duplicate records were related to a SQL join when ordering on a 1-many field.

#### How should this be manually tested?
- create multiple goals with the same properties. Attempt to sort on all goal_note and historical columns (compare with develop)
- This introduces a lot of new changes. Double check all existing functionality. There should be no differences in functionality

#### What are the relevant tickets?
[#4886](https://github.com/SEED-platform/seed/issues/4886)

#### Screenshots (if appropriate)
